### ISSUE TITLE
Add dropdowns to the list

### ISSUE THREAD

**Issue by Anmol270900** [tuvtran/project-based-learning#259]
### Description

I was just wondering if we can add dropdowns to the lists of tutorials. It would make it easier for users to surf through all the technologies.

[
![Screenshot (370)](https://user-images.githubusercontent.com/43845658/94921519-195d7600-04d6-11eb-95b7-f680d9772dbf.png)
](url)

### Is this something you're interested in working on?

- [X] YES
- [ ] NO

**Comment by shreyubee** [comment]
I'd like to work on it

**Comment by siddarthsingh28** [comment]
How can I contribute

**Comment by yungjoeh** [comment]
Yes, I'd love to... But recently, i have been coding with my phone, a friend of mine helped me push  to GitHub

**Comment by Nusart** [comment]
Hi @Anmol270900 , I can contribute to this, please assign me this issue.

**Comment by vinayakdotgupta** [comment]
I would like to help

**Comment by Nirvana404** [comment]
I would like to help u

**Comment by sarav18302** [comment]
I would like to contribute to this , can you explain me what exactly I have to do

**Comment by ShampitaBhattacharjee** [comment]
I can help you with this
